src/Service.Tests/Unittests/ConfigValidationUnitTests.cs

```diff
@@ -2175,15 +2175,15 @@ public void ValidateUniqueRestPathsForEntitiesInConfig(
             DisplayName = "Runtime base-route specified as null for AppService authentication provider passes config validation.")]
         [DataRow(null, "StaticWebApps", false,
             DisplayName = "Runtime base-route specified as null for Static Web Apps authentication provider passes config validation.")]
-        [DataRow("/    ", "StaticWebApps", false,
+        [DataRow("/    ", "StaticWebApps", true, "Runtime base-route contains white spaces.",
```

Comment:
please do not mix tests.

Reply:
since the path contains white spaces in the URI it should fail, hence modified this test

Reply:
was the original test wrong? or did we change some existing functionality?

Reply:
yes modified the functionality to add another validation case

Reply:
This could be a breaking change. Please highlight the same in your PR description.

Reply:
updated the description